plugins/utils/which-key.nix

```diff
@@ -55,257 +265,308 @@ with lib;
+
     plugins = {
-      marks = helpers.defaultNullOpts.mkBool true "shows a list of your marks on ' and `";
+      marks = defaultNullOpts.mkBool true "Shows a list of your marks on ' and `";
 
-      registers = helpers.defaultNullOpts.mkBool true ''shows your registers on " in NORMAL or <C-r> in INSERT mode'';
+      registers = defaultNullOpts.mkBool true ''Shows your registers on " in NORMAL or <C-r> in INSERT mode'';
 
       spelling = {
-        enabled = helpers.defaultNullOpts.mkBool true "enabling this will show WhichKey when pressing z= to select spelling suggestions";
-        suggestions = helpers.defaultNullOpts.mkInt 20 "how many suggestions should be shown in the list?";
+        enabled = defaultNullOpts.mkBool true "Enabling this will show WhichKey when pressing z= to select spelling suggestions";
+        suggestions = defaultNullOpts.mkInt 20 "How many suggestions should be shown in the list?";
       };
 
       presets = {
-        operators = helpers.defaultNullOpts.mkBool true "adds help for operators like d, y, ...";
-        motions = helpers.defaultNullOpts.mkBool true "adds help for motions";
-        textObjects = helpers.defaultNullOpts.mkBool true "help for text objects triggered after entering an operator";
-        windows = helpers.defaultNullOpts.mkBool true "default bindings on <c-w>";
-        nav = helpers.defaultNullOpts.mkBool true "misc bindings to work with windows";
-        z = helpers.defaultNullOpts.mkBool true "bindings for folds, spelling and others prefixed with z";
-        g = helpers.defaultNullOpts.mkBool true "bindings for prefixed with g";
+        operators = defaultNullOpts.mkBool true "Adds help for operators like d, y, ...";
+        motions = defaultNullOpts.mkBool true "Adds help for motions";
+        text_objects = defaultNullOpts.mkBool true "Help for text objects triggered after entering an operator";
+        windows = defaultNullOpts.mkBool true "Default bindings on <c-w>";
+        nav = defaultNullOpts.mkBool true "Misc bindings to work with windows";
+        z = defaultNullOpts.mkBool true "Bindings for folds, spelling and others prefixed with z";
+        g = defaultNullOpts.mkBool true "Bindings for prefixed with g";
       };
     };
 
-    operators = helpers.defaultNullOpts.mkAttrsOf types.str { gc = "Comments"; } ''
-      add operators that will trigger motion and text object completion
-      to enable all native operators, set the preset / operators plugin above
-    '';
+    win = {
+      no_overlap = defaultNullOpts.mkBool true "Don't allow the popup to overlap with the cursor.";
 
-    keyLabels = helpers.defaultNullOpts.mkAttrsOf types.str { } ''
-      override the label used to display some keys. It doesn't effect WK in any other way.
-    '';
+      border = defaultNullOpts.mkBorder "none" "which-key" "";
 
-    motions = {
-      count = helpers.defaultNullOpts.mkBool true "";
-    };
+      padding = defaultNullOpts.mkNullable (with types; listOf int) [
```

Comment:
Is it `int` or `unsignedInt`?

Reply:
I'll double check. I originally assumed it could be used with negative.

Reply:
Can be negative to reduce padding. But, requires 2 number.